Library/Formula/texi2html.rb

```diff
@@ -1,8 +1,8 @@
 class Texi2html < Formula
   desc "Convert TeXinfo files to HTML"
   homepage "http://www.nongnu.org/texi2html/"
-  url "http://download.savannah.gnu.org/releases/texi2html/texi2html-1.82.tar.gz"
-  sha256 "6c7c94a2d88ffe218a33e91118c2b039336cbe3f2f8b4e3a78e4fd1502072936"
+  url "http://download.savannah.gnu.org/releases/texi2html/texi2html-5.0.tar.gz"
+  sha256 "e60edd2a9b8399ca615c6e81e06fa61946ba2f2406c76cd63eb829c91d3a3d7d"
 
   bottle do
     cellar :any_skip_relocation
```

Comment:
Yes. The CI will handle anything changed in the bottle block, including the notation on whether it needs relocation or not.